Watermark On Image/Watermark_on_image.py

```diff
@@ -0,0 +1,28 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+
+path = input("Input the path of the image: ")
+path = path.strip('""')
+im = Image.open(path)
+width, height = im.size
+
+draw = ImageDraw.Draw(im)
+
+#input the text that needs to be printed
+text = input('Enter the text for the watermark: ' )
+font = ImageFont.truetype('arial.ttf', 60 )
+textwidth, textheight = draw.textsize(text, font )
+
+# calculate the x,y coordinates of the text
```

Comment:
Include the options to choose the position of the watermark: Top Right, Top Left, Bottom Right, Bottom Left, and Center.